models/issue_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestIssue_ReplaceLabels(t *testing.T) {
+func TestIssueReplaceLabels(t *testing.T) {
```

Comment:
I thought that the naming conventions for tests matched those of testable examples (https://blog.golang.org/examples)? Was I mistaken?